site/smart-accounts/session-keys/index.md

```diff
@@ -0,0 +1,63 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: Introduction to using Session Keys
+  - - meta
+    - name: description
+      content: Learn about Alchemy's ERC-6900 Compatible Session Key Plugin.
+  - - meta
+    - property: og:description
+      content: Learn about Alchemy's ERC-6900 Compatible Session Key Plugin.
+  - - meta
+    - name: twitter:title
+      content: Introduction to using Session Keys
+  - - meta
+    - name: twitter:description
+      content: Learn about Alchemy's ERC-6900 Compatible Session Key Plugin.
+---
+
+# Session Key Plugin Overview
```

Comment:
we should also add direct links to the ref impl of the session key plugin github, and also the solidity code. (https://github.com/alchemyplatform/modular-account/tree/develop/src/plugins/session )

Reply:
Agree with this, and want to mention that we should link to the v1.0.0 release of the session key plugin rather than the file in the `develop` branch.

Reply:
will follow up in a separate PR with the deployment addresses